### DISCUSSION TITLE
可否关闭双击全屏功能

### DISCUSSION THREAD

**Discussion by Magic-glass** [the1812/Bilibili-Evolved#3544] [功能建议 / Ideas]
电脑双屏，取焦点最方便就是点两下视频，有一段时间这个双击全屏好像是被取消了
现在又加回来了？
这个旧脚本似乎又用不了了https://github.com/exdrcw/CancelBilibiliDoubleClick
这双击全屏真的好恶心啊。。。_(:зゝ∠)_

**Comment by Geeyun-JY3** [comment]
似乎只要装了这个脚本破站本身的双击全屏就用不了了（不确定是否还要安装某个组件）：#2131

3.x 播放器是没适配：#3187

**Reply by Magic-glass** [reply]
> 似乎只要装了这个脚本破站本身的双击全屏就用不了了（不确定是否还要安装某个组件）：#2131
> 
> 3.x 播放器是没适配：#3187

发完这个discussions我搜了一下，才发现https://github.com/the1812/Bilibili-Evolved/discussions/2131 的说法，然后我才去发的issue
_(:зゝ∠)_希望有能力的大佬能下手搞一搞
